.github/workflows/release.yaml

```diff
@@ -35,8 +37,44 @@ jobs:
           java-version: "21"
           distribution: "temurin"
           server-id: github
-      - name: Publish package
-        run: mvn --batch-mode deploy -s settings.xml
+      # stage maven profile
+      - name: Set up JDK to publish to GitHub Packages
+        if: github.ref == 'refs/heads/main'
+        uses: actions/setup-java@5896cecc08fd8a1fbdfaf517e29b571164b031f7
+        with:
+          java-version: "11"
```

Comment:
I think you need a later java version; the project pom has Java 17 as the target; Version 21 is currently used in the build